tutorials/export/exporting_basics.rst

```diff
@@ -239,46 +241,48 @@ changed:
 Setting a main scene
 --------------------
 
```

Comment:
As above this section should go we shouldn't ask user to rewrite our demo game just so they can learn how to export.